makefile

```diff
@@ -87,12 +87,6 @@ version_major: venv
 	bump2version --current-version $(CURRENT_VERSION) major $(VERSIONING_FILES)
 .PHONY: version_major
 
-clean_docs:
-	rm -rf docs/build
-	rm -rf docs/examples
-	rm -rf docs/source/apiref
-	rm -rf docs/source/examples
-.PHONY: clean_docs
```

Comment:
Why do we remove it and not remove its dependencies (doc and clean depend on it).